matchms/exporting/save_spectra.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pickle
+from typing import List
+from matchms.exporting import save_as_json, save_as_mgf, save_as_msp
+from matchms.Spectrum import Spectrum
+from matchms.typing import SpectrumType
+
+
+def save_spectra(spectrums: List[SpectrumType],
+                 file: str) -> None:
```

Comment:
According to #557 the export_style should also be added as a parameter to this function.